examples/java/spring-boot/src/test/java/uk/gov/api/springboot/dtos/MetadataDtoTest.java

```diff
@@ -0,0 +1,117 @@
+package uk.gov.api.springboot.dtos;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import uk.gov.api.springboot.daos.MetadataDao;
+
+class MetadataDtoTest {
+  @Nested
+  class Of {
+
+    @Test
+    void mapsApiVersion() {
+      var dao = getMetadataDao();
+
+      MetadataDto actual = MetadataDto.of(dao);
+
+      assertApiVersion(actual, "api.gov.uk/v1alpha");
+    }
+
+    @Test
+    void mapsName() {
+      var dao = getMetadataDao();
+
+      MetadataDto actual = MetadataDto.of(dao);
+
+      assertName(actual, "name 1");
+    }
+
+    @Test
+    void mapsDescription() {
+      var dao = getMetadataDao();
+
+      MetadataDto actual = MetadataDto.of(dao);
+
+      assertDescription(actual, "description 1");
+    }
+
+    @Test
+    void mapsUrl() {
+      var dao = getMetadataDao();
+
+      MetadataDto actual = MetadataDto.of(dao);
+
+      assertUrl(actual, "https://www.example.foo");
+    }
+
+    @Test
+    void mapsContact() {
+      var dao = getMetadataDao();
+
+      MetadataDto actual = MetadataDto.of(dao);
+
+      assertContact(actual, "contact 1");
+    }
+
+    @Test
+    void mapsOrganisation() {
+      var dao = getMetadataDao();
+
+      MetadataDto actual = MetadataDto.of(dao);
+
+      assertOrganisation(actual, "org 1");
+    }
+
+    @Test
+    void mapsDocumentationUrl() {
+      var dao = getMetadataDao();
+
+      MetadataDto actual = MetadataDto.of(dao);
+
+      assertDocumentationUrl(actual, "https://www.exampledocs.foo");
+    }
+
+    private MetadataDao getMetadataDao() {
+      var dao = new MetadataDao();
+      dao.setName("name 1");
+      dao.setDescription("description 1");
+      dao.setContact("contact 1");
+      dao.setOrganisation("org 1");
+      dao.setDocumentationUrl("https://www.exampledocs.foo");
+      dao.setApiVersion("api.gov.uk/v1alpha");
+      dao.setUrl("https://www.example.foo");
+      return dao;
+    }
+
+    private void assertApiVersion(MetadataDto metadataDto, String apiVersion) {
```

Comment:
I'm not sure what the benefits are of these methods if they're one-liners only used in one place, or is it just to make the tests more readable?